src/voxelalgorithms.cpp

```diff
@@ -1055,10 +1047,9 @@ bool propagate_block_sunlight(Map *map, INodeDefManager *ndef,
 					modified = true;
 					relight->push(LIGHT_SUN, current_pos, data->target_block,
 						block, 4);
-				} else {
+				} else
```

Comment:
This should be reverted to how it was before, we don't mix bracketed and non-bracketed style in one conditional.